src/parsers/css/csstree.js

```diff
@@ -0,0 +1,38 @@
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from 'css-tree/package.json';
+
+const ID = 'csstree';
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: pkg.homepage || 'https://github.com/csstree/csstree',
+  locationProps: new Set(['loc']),
+
+  loadParser(callback) {
+    require(['css-tree'], callback);
+  },
+
+  parse(csstree, code) {
+    return JSON.parse(JSON.stringify(csstree.parse(code, {
```

Comment:
Updated `csstree` to new version with `toPlainObject()` method on board. Now it looks better ;)